src/context/MovieModalContext.tsx

```diff
@@ -9,12 +10,16 @@ interface MovieModalContextType {
 
 export const MovieModalContext = React.createContext<MovieModalContextType>({
   isOpen: false,
-  setIsOpen: () => {},
-  handleOpen: () => {},
+  setIsOpen: noop,
+  handleOpen: noop,
```

Comment:
```suggestion
  setIsOpen: noop(),
  handleOpen: noop(),
```

You have to call it so it will return the function.